zvariant/src/file_path.rs

```diff
@@ -0,0 +1,106 @@
+use std::{ffi::{OsStr, OsString}, path::{Path, PathBuf}};
+
+use serde::{de::{self, Visitor}, Deserialize, Deserializer, Serialize, Serializer};
+use crate::de::Deserializer as ZbusDeserializer;
```

Comment:
We should not need to use a specific deserializer here and I don't see any use of this anyway.